deepmd/pt/model/task/fitting.py

```diff
@@ -419,6 +415,8 @@ def __setitem__(self, key, value):
             self.aparam_inv_std = value
         elif key in ["scale"]:
             self.scale = value
+        elif key in ["constant_matrix"]:
+            self.constant_matrix = value
```

Comment:
the variable should be maintained by the polarizability_fitting, not by the general fitting.